components/script/dom/bindings/trace.rs

```diff
@@ -881,6 +882,58 @@ where
     }
 }
 
+unsafe impl<J> JSTraceable for Hand<J>
```

Comment:
Hmm, it would be nice if we could derive this, it'll be very easy to mess this up down the road :(

Reply:
Can't without introducing a mozjs dep to webxr